assets/styles/style.css

```diff
@@ -1,10 +1,23 @@
-@import url('https://fonts.googleapis.com/css?family=Kavivanar|Merriweather:300,300i,400,400i,700,700i,900,900i|Source+Code+Pro:200,300,400,500,600,700,900|Open+Sans+Condensed:300,300i,700');
+@import url('https://fonts.googleapis.com/css?family=Roboto|Merriweather:300,300i,400,400i,700,700i,900,900i|Source+Code+Pro:200,300,400,500,600,700,900|Open+Sans+Condensed:300,300i,700');
+
+:root {
+  --main-bg-color: #FFFBE7;
+  --main-txt-color: #0476A3;
+  --main-txt-color-dark: #026494;
+  --bg-highlight-color: #FFF5C2;
+  --secondary-bg-color: #0476A3;
+  --secondary-txt-color: #ffffff;
+  --main-font: 'Merriweather', serif;
+  --secondary-font: 'Roboto', sans-serif;
+  --box-shadow-color: rgba(19, 140, 171, 0.8);
+  --secondary-box-shadow-color: rgba(241, 241, 241, 0.8);
+}
```

Comment:
:raised_hands: I really like that you have named all of these values and moved them to a single place where they can be managed.